Project.toml

```diff
@@ -17,6 +17,7 @@ MsgPack = "99f44e22-a591-53d1-9472-aa23ef4bd671"
 Pkg = "44cfe95a-1eb2-52ea-b672-e2afdf69b78f"
 REPL = "3fa0cd96-eef1-5676-8a61-b3b8758bbffb"
 Sockets = "6462fe0b-24de-5631-8697-dd941f90decc"
+TableIOInterface = "d1efa939-5518-4425-949f-ab857e148477"
```

Comment:
@fonsp do you mean this [deps] section?

Reply:
ah i meant `[compat]`

Reply:
I suggest to add
`TableIOInterface = "0.1"`
to the `[compat]` section.